src/content/docs/developer-tools/kinde-api/add-a-m2m-application-for-api-access.mdx

```diff
@@ -35,6 +39,16 @@ For example, you need a machine to machine application for connecting to [Kinde
 6. Select **Save**
 7. Next: [Get an access token for Kinde’s management API](/developer-tools/kinde-api/get-access-token-for-connecting-securely-to-kindes-api/).
 
+## Change or add scopes to an application accessing the Kinde Management API
+
+Follow this procedure if you already have an application and you experience a scope error or want to add scopes to an application.
```

Comment:
Do you think we should mention we recommend using as few scopes as you need for maximum security?